internal/corazawaf/waf.go

```diff
@@ -420,9 +420,15 @@ func (w *WAF) SetDebugLogPath(path string) error {
 		w.Logger.SetOutput(io.Discard)
 		return nil
 	}
+
+	if path == "/dev/stdout" {
```

Comment:
I understand a lot of current usage is for stdout, but I think it's too conventional to treat stdout / stderr as siblings to skip the latter here. Also IMO, stderr seems more appropriate for debug logs anyways